non-msrv-tests/tests/ui/item_skip.rs

```diff
@@ -1,59 +1,47 @@
-use std::marker::PhantomData;
+use derive_where::derive_where;
```

Comment:
Lol, again, how did you get away with deleting `use std::marker::PhantomData;`, magic.